README.md

```diff
@@ -7,6 +7,12 @@ A simple, responsive lightbox component for displaying an array of images.
 
 ## Quick start
 
+# npm i react-images --save 
+
+### or 
+
+# yarn add react-images
+
```

Comment:
That's a good idea, but I think it may be a bit intense to use headings - maybe something like:

```bash
npm install react-images --save
```
or 
```bash
yarn add react-images
```

* * *

Also beginners are more likely to recognise `install` rather than `i` for the npm script. What do you think?